internal/convert.go

```diff
@@ -201,6 +203,11 @@ func (conv *Conv) Rows() int64 {
 	return n
 }
 
+// GetRows return count of data rows for a srcTable
+func (conv *Conv) GetRows(srcTable string) int64 {
```

Comment:
In general, we've avoided adding accessors to convert.go, and instead have directly accessed the subfields of conv as needed. Note that there are functions like BadRows and Statements, but they all do some computation e.g. sum up bad rows etc.

For consistency I think we should continue down that path. That means we'd have to make stats public. Given that much of Conv is being made public anyway, that seems reasonable.

In other words, if anything in convert.go would need an accessor, let's just make it public and avoid the accessor. Same comment for the other GetNNN functions added below.

Hengfeng -- do you have any thoughts on this?

Reply:
Ok, I have made stats and it's fields public and removed accessors i.e GetNNN functions.